packages/react/src/react/views/collections/list/list.client.tsx

```diff
@@ -178,7 +178,7 @@ export function ClientCollectionListView() {
         table={table}
         loadingItems={pagination.pageSize}
         className="static"
-        onRowClick="toggleSelect"
+        onRowClick={listViewProps.actions?.select ? 'toggleSelect' : undefined}
```

Comment:
same as previous comment

```suggestion
        onRowClick={(listViewProps.actions?.select || listViewProps.actions?.delete) ? 'toggleSelect' : undefined}
```

Reply:
same as previous